test/e2e/specs/default-config/recordedit/submission-disabled.spec.js

```diff
@@ -5,15 +5,16 @@ var testParams = {
         name: "id",
```

Comment:
In the `errors.spec.js`, it seems like we had planed to test this duplicate case and there's a `uniqueConstraint` message defined for it, which is not used at all.  We also could completely move this spec to the errors spec so that all the error-related test cases are in one place.

These are just observations. I don't think you should do these changes now and these can be done as part of refactoring for other features.

Reply:
I noticed the same thing but didn't want to include it in the error spec because it was a different "type" of error. This is an alert notification rather than a popup error, so I wanted to keep those tests together for `recordedit` but separate from general error popup errors.